repos/system_upgrade/common/actors/scansourcefiles/tests/unit_test_scansourcefiles.py

```diff
@@ -1,5 +1,175 @@
-def test_scansourcefiles():
-    # TODO(pstodulk): keeping unit tests for later after I check the idea
-    # of this actor with the team.
-    # JIRA: OAMG-10367
-    pass
+import os
+
+import pytest
+
+from leapp.libraries.actor import scansourcefiles
+from leapp.libraries.common import testutils
+from leapp.libraries.stdlib import api, CalledProcessError
+from leapp.models import FileInfo, TrackedFilesInfoSource
+
+# values in list should correspond with the return values of get_source_major_version()
+RHEL_MAJOR_VERSIONS_LIST = list(range(8, 9+1))
```

Comment:
why? what is value in comparison to do just this?

```suggestion
RHEL_MAJOR_VERSIONS_LIST = [8, 9]
```

Also, it's incorrect as `get_source_major_version` returns string, not integer. so correctly it would be:
```suggestion
RHEL_MAJOR_VERSIONS_LIST = ['8', '9']
```

Reply:
Done.